lib/main.dart

```diff
@@ -77,7 +80,15 @@ class MyApp extends StatelessWidget {
                           crossAxisAlignment: CrossAxisAlignment.stretch,
                           children: [
                             TextButton(
-                              onPressed: () => {},
+                              onPressed: () {
+                                // Navigator.of(context).push(MaterialPageRoute(
+                                //     builder: (context) => const SecondRoute()));
+                                Navigator.push(
```

Comment:
The reason you get error 'Navigator operation requested with a context that does not include a Navigator.' is because your `BuildContext` here belongs to class `MyApp`, which is a StatelessWidget. The `build` method of this `MyApp` class will build a MaterialApp, but the `BuildContext` that passed into this build method (in line 15) does not have MaterialApp in its tree yet. 

There are 2 ways you can fix this:
1. Bring the whole Scaffold (from line 24) to a separate StatelessWidget, which will create a sub-tree with new BuildContext, and this BuildContext will include all features that MaterialApp supports
2. Move Scaffold (line 24) into a `Builder` widget, same reason as above.

Reply:
I recommend the first option, the code will look cleaner.